app/celer-export-geant.cc

```diff
@@ -105,7 +105,7 @@ void run(std::string const& gdml_filename,
         ImportDataTrimmer::Input options;
         options.mupp = true;
         ImportDataTrimmer trim(options);
-        trim(&imported);
+        trim(imported);
```

Comment:
Why the change from pointers to references (since our style is typically to pass pointers instead of mutable references when a value is going to be modified)?

Reply:
OK I should've talked with the group about this first... I think the *idea* of annotating arguments to be modified is good when it could be ambiguous, but I increasingly think pointers are the wrong way to do it. Google somewhat recently changed their style guide to use references instead of pointers (except for optional in/out or out-only values), and the standard library and friends don't generally use raw pointers like that. So I think we should remove that guideline from our code style and start think about using references instead... perhaps using some clang libtooling to help refactor our code?

Reply:
Sure, that seems sensible to me. Maybe we can add that as a todo in one of our issues?